tests/h/search/index_test.py

```diff
@@ -98,9 +98,18 @@ def test_index_indexes_all_annotations_to_es(self, db_session, indexer, matchers
 
         indexer.index()
 
-        streaming_bulk.assert_called_once_with(
-            indexer.es_client.conn, matchers.iterable_with([ann_1, ann_2]),
-            chunk_size=mock.ANY, raise_on_error=False, expand_action_callback=mock.ANY)
+        assert streaming_bulk.call_count == 1
```

Comment:
Rather than having lots of asserts, I think you could use `mock.ANY` as the matcher for the second argument and then add just one more assert: `assert list(streaming_bulk.call_args[0][1]) == [ann_1, ann_2]`. The downside of having many asserts is that as soon as one fails, details about the rest are missing.